common/assets/scss/overrides/_visibility.scss

```diff
@@ -13,3 +13,19 @@
     display: block;
   }
 }
+
+@media (min-width: 40.0625em) {
```

Comment:
Maybe use the GOV.UK Design System mixins here too?

Reply:
👍